src/platform/OpenThread/GenericThreadStackManagerImpl_OpenThread.cpp

```diff
@@ -813,10 +813,6 @@ CHIP_ERROR GenericThreadStackManagerImpl_OpenThread<ImplClass>::DoInit(otInstanc
         VerifyOrExit(otInst != NULL, err = MapOpenThreadError(OT_ERROR_FAILED));
     }
 
-#if !defined(__ZEPHYR__) && !defined(ENABLE_CHIP_SHELL) && !defined(PW_RPC_ENABLED)
```

Comment:
The path towards OT CLI interface get's removed here. What will be the substitution to access the OT infrastructure for debugging purposes ?
Noted removing the CLI frees up quite some RAM/flash if no longer required

Reply:
I removed these lines because otCliUartInit API is not available anymore in openthread since openthread/openthread#6243 (comment).
I will push a change such that the user will be able to use a GN build argument for enabling/disabling the OT CLI.

OT CLI takes around 25K of flash with everything compiled with size optimizations - I wonder why it requires so much space.